CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing Guidelines
+
+NOTE: Contribution implies licensing under the terms of [LICENSE]
+(AGPL-3).
+
+## Pull Request Checklist
+
+- Confirm the PR is against the `beta` branch.
+- Ensure commit messages follow the [conventional commits] style.
+- Ensure commit messages adequately explain the reasons for the changes.
+  outlined in the README.
```

Comment:
`outlined in the README` misplaced from somewhere?

Reply:
fixed in [8d685c4](https://github.com/ardriveapp/turbo-sdk/pull/36/commits/8d685c42e495d9f44f6a4077f02ace6d1313283f)